platform_plugin_forum_email_notifier/apps.py

```diff
@@ -11,3 +11,23 @@ class PlatformPluginForumEmailNotifierConfig(AppConfig):
     """
 
     name = "platform_plugin_forum_email_notifier"
+
+    plugin_app = {
+        "settings_config": {
+            "lms.djangoapp": {
+                "common": {"relative_path": "settings.common"},
+                "test": {"relative_path": "settings.test"},
+                "production": {"relative_path": "settings.production"},
+            },
+            "cms.djangoapp": {
+                "common": {"relative_path": "settings.common"},
+                "test": {"relative_path": "settings.test"},
+                "production": {"relative_path": "settings.production"},
+            },
+        },
+    }
+
+    def ready(self):
+        super().ready()
+
+        from platform_plugin_forum_email_notifier import email, tasks, handlers  # pylint: disable=unused-import, import-outside-toplevel
```

Comment:
why are we registering the email module? is it necessary?

Reply:
For hot reloading